src/java.base/share/classes/java/lang/Object.java

```diff
@@ -95,13 +114,26 @@ public Object() {}
      * As far as is reasonably practical, the {@code hashCode} method defined
      * by class {@code Object} returns distinct integers for distinct objects.
      *
+     * <p>This method should avoid throwing or propagating any
+     * exceptions unless it legitimately <em>cannot</em> adhere to its
```

Comment:
should we use terms like "recoverable exceptions" to distinguish things like exceptions from Integer.parseInt?